### ISSUE TITLE
Latest icons missing

### ISSUE THREAD

**Issue by danknotdank** [ianmiller347/feather-icons-react#1]
I noticed there are a few icons missing from more recent updates to the feather library, I was hoping you'd be willing and able to update this, as I prefer this library over [react-feather](https://github.com/carmelopullara/react-feather).

I wanted to make the changes myself and then submit a PR but I think I'm misunderstanding the build process. 

In case it's relevant here's the missing icons:
```json
{"missing": ["archive", "arrow-down-circle", "arrow-left-circle", "arrow-right-circle", "arrow-up-circle", "book-open", "code", "database", "dollar-sign", "folder-minus", "folder-plus", "gift", "git-branch", "git-commit", "git-merge", "git-pull-request", "hard-drive", "linkedin", "rss", "send", "shield-off", "shopping-bag", "terminal", "truck", "youtube", "zap-off"]}
```

Any help is appreciated!

**Comment by ianmiller347** [comment]
Hey! Thank you for opening this issue. I do plan to update this very soon as I was working on a way to automate the updates. Essentially I want to use the json generator from the feather repo and always have the latest icons. I have been working on this locally but haven't pushed up any changes yet. If you have any suggestions on helping implement this that would also help! If not I will try to get these updated within the next week or two.

Thanks!

**Comment by actuallyDan** [comment]
Great, I can try to take a crack at it, but I expect you'll have a much better solution. I'll consider this closed so you can close it whenever you push the changes, or you can close it now and I'll just keep an eye out. Thanks again!

**Comment by ianmiller347** [comment]
@danknotdank this is fixed here: https://github.com/ianmiller347/feather-icons-react/pull/2

published and fixed in latest feather-icons-react version 0.2.0
